api/peripheral_api/routes/Printer.js

```diff
@@ -73,19 +81,32 @@ router.post('/sendPrintRequest', async (req, res) => {
 
   const accountId = ACCOUNT_ID;
   const queueName = PAPER_PRINTING_QUEUE_NAME;
-
-  const sqsParams = {
-    MessageBody: JSON.stringify({
-      location: response.Location,
-      fileNo: fileName,
-      copies: copies,
-      pageRanges,
-    }),
-    QueueUrl: `https://sqs.us-west-2.amazonaws.com/${accountId}/${queueName}`
-  };
-  sqs.sendMessage(sqsParams, (err, data) => {
-    return res.sendStatus(OK);
+  const data = JSON.stringify({
+    location: response.Location,
+    fileNo: fileName,
+    copies: copies,
+    pageRanges,
   });
+  const result  = SqsHandler.pushMessageToQueue(data);
```

Comment:
lint this file one time before merging

```suggestion
  const result = SqsHandler.pushMessageToQueue(data);
```